### ISSUE TITLE
New Feature: Export as SVG

### ISSUE THREAD

**Issue by LukeTimeWalker** [glimpse-editor/Glimpse#168]
Similar to Inkscape, there should be an option "Export as [SVG (.svg)](https://de.wikipedia.org/wiki/Scalable_Vector_Graphics)" where the graphics created in Glimpse are saved as scalable vector graphics.

**Comment by Member1221** [comment]
I don’t think we will ever add this. Glimpse is a raster graphics application and doesn’t have the proper tools or setup internally to handle vector graphics. We’d only be able to make (inaccurate) approximations of SVGs which wouldn’t look good scaled up.

**Comment by TrechNex** [comment]
It is technically possible, but we would have to implement this ourselves as this is not provided upstream.

I've labeled it as an enhancement as it's a feature idea we could pick up later if we learn something new that makes it feasible for us to take it on.

**Comment by jgamble** [comment]
You'd have to have a process to trace the bitmap. Inkscape does this (Path->Trace Bitmap, for those looking for the menu item), but credits its code to Potrace, by Peter Selinger.

[Potrace on sourceforge](http://potrace.sourceforge.net/).

**Comment by LukeTimeWalker** [comment]
It would already be an improvement if you could export images you created in Glimpse (for example using the free/rectangle/ellipse selection tool) as SVG. This would make it relatively easy to vectorize simple raster images.

**Comment by Member1221** [comment]
I will move this to a 1.0 candidate; once we hard fork it'd be a lot more feasible adding such a large feature; then.
